src/main/java/rx/Observable.java

```diff
@@ -62,10 +62,72 @@ protected Observable(OnSubscribe<T> f) {
     }
 
     /**
-     * <strong>This method requires advanced knowledge about building operators and data sources; please consider
-     * other standard methods first, such as {@link Observable#fromEmitter(Action1, Emitter.BackpressureMode)};</strong>
-     * Returns an Observable that will execute the specified function when a {@link Subscriber} subscribes to
-     * it.
+     * Constructs an Observable in an unsafe manner, that is, unsubscription and backpressure handling
+     * is the responsibility of the OnSubscribe implementation.
+     * @param <T> the value type emitted
+     * @param f the callback to execute for each individual Subscriber that subscribes to the
+     *          returned Observable
+     * @return the new Observable instance
+     * @deprecated 1.2.7 - inherently unsafe, use the other create() methods for basic cases or
+     * see {@link #unsafeCreate(OnSubscribe)} for advanced cases (such as custom operators)
+     * @see #create(SyncOnSubscribe)
+     * @see #create(AsyncOnSubscribe)
+     * @see #create(Action1, rx.Emitter.BackpressureMode)
+     */
+    @Deprecated
+    public static <T> Observable<T> create(OnSubscribe<T> f) {
+        return new Observable<T>(RxJavaHooks.onCreate(f));
+    }
+
+    /**
+     * Provides an API (via a cold Observable) that bridges the reactive world with the callback-style,
+     * generally non-backpressured world.
+     * <p>
+     * Example:
+     * <pre><code>
+     * Observable.&lt;Event&gt;create(emitter -&gt; {
+     *     Callback listener = new Callback() {
+     *         &#64;Override
+     *         public void onEvent(Event e) {
+     *             emitter.onNext(e);
+     *             if (e.isLast()) {
+     *                 emitter.onCompleted();
+     *             }
+     *         }
+     *
+     *         &#64;Override
+     *         public void onFailure(Exception e) {
+     *             emitter.onError(e);
+     *         }
+     *     };
+     *
+     *     AutoCloseable c = api.someMethod(listener);
+     *
+     *     emitter.setCancellation(c::close);
+     *
+     * }, BackpressureMode.BUFFER);
+     * </code></pre>
+     * <p>
+     * You should call the Emitter's onNext, onError and onCompleted methods in a serialized fashion. The
+     * rest of its methods are thread-safe.
+     *
+     * @param <T> the element type
+     * @param emitter the emitter that is called when a Subscriber subscribes to the returned {@code Observable}
+     * @param backpressure the backpressure mode to apply if the downstream Subscriber doesn't request (fast) enough
+     * @return the new Observable instance
+     * @see Emitter
+     * @see Emitter.BackpressureMode
+     * @see rx.functions.Cancellable
+     * @since 1.2.7 - experimental
+     */
+    @Experimental
```

Comment:
The lack of a stable `fromEmitter` and now two-argument `create()` is quite a burden on libraries. I would love to see this go to stable in 1.3 (skipping `@Beta`).